buildscripts/condarecipe.local/meta.yaml

```diff
@@ -53,6 +53,10 @@ requirements:
     - tbb >=2021    # [not (armv6l or armv7l or aarch64 or linux32 or ppc64le)]
     # avoid confusion from openblas bugs
     - libopenblas !=0.3.6      # [x86_64]
+    # 0.3.17 buggy on M1 silicon
+    # https://github.com/xianyi/OpenBLAS/blob/develop/Changelog.txt#L118
```

Comment:
This reference isn't stable, it refers to the `develop` branch?

Reply:
Good catch, thank you.